io.github.thiefmd.themegenerator.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id":"io.github.thiefmd.themegenerator",
+    "runtime":"org.gnome.Platform",
+    "runtime-version":"3.38",
+    "sdk":"org.gnome.Sdk",
+    "command":"io.github.thiefmd.themegenerator",
+    "finish-args":[
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=x11"
```

Comment:
It's either `x11` alone or `fallback-x11` with `wayland`. And suspect the latter is probably what you want.